apps/public-docsite-v9/.eslintrc.json

```diff
@@ -2,6 +2,7 @@
   "extends": ["plugin:@fluentui/eslint-plugin/react--legacy"],
   "root": true,
   "rules": {
+    "@griffel/styles-file": "off",
```

Comment:
We don't publish this package, no sense to apply the rule there.

Reply:
we dont publish it but  its an app that is deployed and customer facing, so maybe we should reconsider this approach ?

Reply:
Done in 4195b00 👍